package.json

```diff
@@ -33,23 +33,20 @@
     "@nextcloud/router": "^2.1.2",
     "@nextcloud/vue": "^7.12.1",
     "attachmediastream": "^2.1.0",
-    "color.js": "^1.2.0",
     "crypto-js": "^4.1.1",
     "debounce": "^1.2.1",
     "emoji-mart-vue-fast": "^15.0.0",
     "emoji-regex": "^10.2.1",
     "escape-html": "^1.0.3",
     "extendable-media-recorder": "^7.1.20",
     "extendable-media-recorder-wav-encoder": "^7.0.93",
-    "extendable-media-recorder-wav-encoder-worker": "^8.0.85",
```

Comment:
This is not removed from the deps and still used.
I assume we had to bind it to a specific version for some reason. Maybe @danxuliu remembers?

Reply:
I don't mind to wait for it and resolve the question.

This is from my observation:
We are using `connect` function from this module: https://github.com/chrisguttandin/extendable-media-recorder-wav-encoder/blob/cd20c8b96fad327ef4b2aafad4a4d953755f577a/src/module.ts#L12
It uses minified version of worker package from its own dependencies: https://github.com/chrisguttandin/extendable-media-recorder-wav-encoder/blob/cd20c8b96fad327ef4b2aafad4a4d953755f577a/src/worker/worker.ts, https://github.com/search?q=repo%3Achrisguttandin%2Fextendable-media-recorder-wav-encoder%20extendable-media-recorder-wav-encoder-worker&type=code

Reply:
It must be safe to remove.

We don't use it directly at all, it is only used by `extendable-media-recorder-wav-encoder`.
Moreover, the only work of `extendable-media-recorder-wav-encoder` is load end expose worker's methods.

And `extendable-media-recorder-wav-encoder` has the same version as listed here, so it doesn't affect resolved version at all.

Reply:
> This is not removed from the deps and still used. I assume we had to bind it to a specific version for some reason. Maybe @danxuliu remembers?

No idea, sorry, and the commit/pull request where [it was introduced does not say much either](https://github.com/nextcloud/spreed/commit/63fe1ac0d71cd660bee3df5ff3d304d5aca4871e#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R42) :shrug: But in any case I am late to the party and it was removed already, so... :-P